MontyHallKata/Models/DoorsFactory.cs

```diff
@@ -0,0 +1,40 @@
+using MontyHallKata.Models.Entity;
+
+namespace MontyHallKata.Models
+{
+    public static class DoorsFactory
+    {
```

Comment:
Little bit confused with the Factory, since my understanding of a factory is someplace creating objects but not providing functions.
Could move these static methods to `Door` class.

ref: https://www.dofactory.com/net/factory-method-design-pattern